### PR TITLE
Mdlrc doc example

### PR DESCRIPTION
**Pull request by Antares1980**
## Description
I could not find a good mdlrc example to modify current rules and styles. This Pull Request is adding an example of .mdlrc  and style files.

## Related Issues
Solution was found in a closed Issue #199 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x ] Documentation (non-breaking change that does not add functionality but updates documentation)
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

## Checklist:

- [x ] I have read the [**CONTRIBUTING**](https://github.com/markdownlint/markdownlint/blob/master/CONTRIBUTING.md) document.
- [x ] Wrote [good commit messages](https://chris.beams.io/posts/git-commit/)
- [x ] Feature branch is up-to-date with `master`, if not - rebase it
- [x ] Added tests for all new/changed functionality, including tests for positive and negative scenarios
- [x ] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences

### CODE REVIEW

**Review by jaymzh — CHANGES_REQUESTED** [review]
Thanks!!

This information is in the wiki, but we get lots of questions about it, so it's probably better here.

File: example/new_style_example.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/ruby
```
[the remark this answers is not in the document]
Comment (Antares1980) [reply example/new_style_example.rb]: Thanks for the comments. I accepted your suggestion adding the changes to the file.

[the remark this answers is not in the document]
Comment (jaymzh) [reply example/new_style_example.rb]: You still have the shebang line here, can you please remove it?

[the remark this answers is not in the document]
Comment (Antares1980) [reply example/new_style_example.rb]: Hi,

Shebang line removed from file.

**Review by jaymzh — APPROVED** [review]
Nice. Thanks so much for contributing!

**Pull request merged**